analyzers/src/SonarAnalyzer.CSharp/Rules/InsteadOfAny.cs

```diff
@@ -25,6 +25,17 @@ public sealed class InsteadOfAny : InsteadOfAnyBase<SyntaxKind, InvocationExpres
 {
     protected override ILanguageFacade<SyntaxKind> Language => CSharpFacade.Instance;
 
+    protected override HashSet<SyntaxKind> ExitParentKinds => new()
```

Comment:
this will generate a HashSet at each access. This is why I mentioned [here](https://github.com/SonarSource/sonar-dotnet/pull/7332#discussion_r1214375717) to return the field. 

But it's easier to
```
protected override HashSet<SyntaxKind> ExitParentKinds { get; } = new()
...
```

and that will also solve the problem

Reply:
Ah, very good point. Quite easy to get it wrong.